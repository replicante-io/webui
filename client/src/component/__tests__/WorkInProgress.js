import React from 'react';
import renderer from 'react-test-renderer';

import WorkInProgress from '../WorkInProgress';


describe('WorkInProgress', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<WorkInProgress />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
