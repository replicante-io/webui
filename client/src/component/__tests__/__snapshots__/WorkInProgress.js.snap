// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkInProgress renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col"
  >
    <div
      className="card text-center border-warning"
    >
      <div
        className="card-header border-warning"
      >
        Work In Progress
      </div>
      <div
        className="card-body"
      >
        <h5
          className="card-title"
        >
          Thank you for checking out 
          <a
            href="https://www.replicante.io/"
          >
            Replicante
          </a>
          !
        </h5>
        <p
          className="card-text"
        >
          It seems you sumbled across a feature that is not yet implemented.
        </p>
        <p
          className="card-text large-icon"
        >
          <i
            className="fas fa-wrench"
          />
        </p>
        <p
          className="card-text"
        >
          We are working hard to implement lots of awesome features so hang tight!
        </p>
      </div>
    </div>
  </div>
</div>
`;
